cosmos/operators/azure_container_instance.py

```diff
@@ -0,0 +1,185 @@
+        self.build_command(context, cmd_flags)
+        self.log.info(f"Running command: {self.command}")
+        result = int(super().execute(context))
+        logger.info(result)
+        return result
+
+    def build_command(self, context: Context, cmd_flags: list[str] | None = None) -> None:
+        # For the first round, we're going to assume that the command is dbt
+        # This means that we don't have openlineage support, but we will create a ticket
+        # to add that in the future
+        self.dbt_executable_path = "dbt"
+        dbt_cmd, env_vars = self.build_cmd(context=context, cmd_flags=cmd_flags)
+        self.command: list[str] = dbt_cmd
+
+    def execute(self, context: Context) -> int:
+        return self.build_and_run_cmd(context=context)
+
+
+class DbtLSAzureContainerInstanceOperator(DbtAzureContainerInstanceBaseOperator):
+    """
+    Executes a dbt core ls command.
+    """
+
+    ui_color = "#DBCDF6"
+
+    def __init__(self, **kwargs: Any) -> None:
+        super().__init__(**kwargs)
+        self.base_cmd = ["ls"]
+
+
+class DbtSeedAzureContainerInstanceOperator(DbtAzureContainerInstanceBaseOperator):
+    """
+    Executes a dbt core seed command.
+
+    :param full_refresh: dbt optional arg - dbt will treat incremental models as table models
+    """
+
+    ui_color = "#F58D7E"
+
+    def __init__(self, full_refresh: bool = False, **kwargs: Any) -> None:
+        self.full_refresh = full_refresh
+        super().__init__(**kwargs)
+        self.base_cmd = ["seed"]
+
+    def add_cmd_flags(self) -> list[str]:
+        flags = []
+        if self.full_refresh is True:
+            flags.append("--full-refresh")
+
+        return flags
+
+    def execute(self, context: Context) -> int:
+        cmd_flags = self.add_cmd_flags()
+        return self.build_and_run_cmd(context=context, cmd_flags=cmd_flags)
+
+
+class DbtSnapshotAzureContainerInstanceOperator(DbtAzureContainerInstanceBaseOperator):
+    """
+    Executes a dbt core snapshot command.
+
+    """
+
+    ui_color = "#964B00"
+
+    def __init__(self, **kwargs: Any) -> None:
+        super().__init__(**kwargs)
+        self.base_cmd = ["snapshot"]
+
+
+class DbtRunAzureContainerInstanceOperator(DbtAzureContainerInstanceBaseOperator):
+    """
+    Executes a dbt core run command.
+    """
+
+    ui_color = "#7352BA"
+    ui_fgcolor = "#F4F2FC"
+
+    def __init__(self, **kwargs: Any) -> None:
+        super().__init__(**kwargs)
+        self.base_cmd = ["run"]
+
+
+class DbtTestAzureContainerInstanceOperator(DbtAzureContainerInstanceBaseOperator):
+    """
+    Executes a dbt core test command.
+    """
+
+    ui_color = "#8194E0"
+
+    def __init__(self, on_warning_callback: Callable[..., Any] | None = None, **kwargs: Any) -> None:
+        super().__init__(**kwargs)
+        self.base_cmd = ["test"]
+        # as of now, on_warning_callback in docker executor does nothing
```

Comment:
nit: this comment is from the docker operator

Reply:
Removed it.